src/logger/index.js

```diff
@@ -13,31 +17,31 @@ const alignedWithColorsAndTime = combine(colorize(),
     return `${timestamp} ${level.padEnd(15)} ${prefix.padEnd(55)} ${message}`;
   }));
 
-const logger = winston.createLogger({
-  format: alignedWithColorsAndTime,
-  transports: [
-    new winston.transports.Console({
-      handleExceptions: true
-    }),
-  ],
-  exitOnError: false
-});
+const consoleTransport = new winston.transports.Console();
 
-export function onStartTrackingChanges() {
-  logger.info({ message: 'Start tracking changes…' });
-}
+const transports = [ consoleTransport ];
 
-export function onEndTrackingChanges() {
-  logger.info({ message: 'Tracking changes done.' });
-}
+if (config.get('logger.sendMailOnError')) {
+  const mailErrorTransport = new winston.transports.Mail({
+    level: 'error',
+    to: config.get('logger.sendMailOnError.to'),
+    from: config.get('logger.sendMailOnError.from'),
+    subject: 'CGUs - Error Report',
```

Comment:
More common in email subjects.

```suggestion
    subject: '[CGUs] Error Report',
```